src/Utils/kernel_launching.jl

```diff
@@ -83,6 +86,12 @@ KernelParameters(args::Tuple) = KernelParameters(args...)
 contiguousrange(range::NTuple{N, Int}, offset::NTuple{N, Int}) where N = Tuple(1+o:r+o for (r, o) in zip(range, offset))
 flatten_reduced_dimensions(worksize, dims) = Tuple(d ∈ dims ? 1 : worksize[d] for d = 1:3)
 
+# Internal utility to launch a function mapped on an index_map
```

Comment:
Might make sense to write a longer comment here. `index_map` is the key feature that is utilized by active_cells_map, right?